File: src/main/java/training/katas/exes_and_ohs/exes_and_ohs.rb
def XO(str)
    #your code here
    lower_cased = str.downcase
    lower_cased.split('').select { |chr| chr == 'x' }.length == lower_cased.split('').select { |chr| chr == 'o' }.length
end

puts XO('aaa')
puts XO('xo')
puts XO('XO')
puts XO('xo0')
puts XO('xxxoo')
puts XO("xxOo")

File: src/main/java/training/katas/exes_and_ohs/ExesAndOhs.dart
bool XO(str) {
  // your code here
  var lowerCased = str.toLowerCase();
  return lowerCased.runes.where((int i) {
    return i == 'x'.codeUnitAt(0);
  }).length == lowerCased.runes.where((int i) {
    return i == 'o'.codeUnitAt(0);
  }).length;
}

void main() {
  print(XO("xox"));
}
